src/test/java/io/axoniq/demo/university/faculty/write/changecoursecapacity/ChangeCourseCapacityAxonFixtureTest.java

```diff
@@ -0,0 +1,78 @@
+package io.axoniq.demo.university.faculty.write.changecoursecapacity;
+
+import io.axoniq.demo.university.faculty.FacultyAxonTestFixture;
+import io.axoniq.demo.university.faculty.events.CourseCapacityChanged;
+import io.axoniq.demo.university.faculty.events.CourseCreated;
+import io.axoniq.demo.university.faculty.events.CourseRenamed;
+import io.axoniq.demo.university.shared.ids.CourseId;
+import org.axonframework.test.fixture.AxonTestFixture;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class ChangeCourseCapacityAxonFixtureTest {
+
+    private AxonTestFixture fixture;
+
+    @BeforeEach
+    void beforeEach() {
+        fixture = FacultyAxonTestFixture.slice(ChangeCourseCapacityConfiguration::configure);
+    }
+
+    @Test
+    void givenNotExistingCourse_WhenChangeCapacity_ThenException() {
```

Comment:
Ow, so command handler do still work, as long as it's the `FacultyAxonTestFixture`! Is that reflected in the getting started docs?